cmd/main_test.go

```diff
@@ -75,6 +75,10 @@ func TestMain(t *testing.T) {
 			name: "collect-and-analyze",
 			args: "--resource-dump-file examples/output/resources2.json --filename examples/output/analysis2.txt",
 		},
+		{
+			name: "collect-and-synthesize",
+			args: "--synthesize-dump-dir examples/output/synthesis",
```

Comment:
no need for input resources file?

Reply:
no, input file is optional, in this test it collect the resources.

Reply:
but it cannot run on the ci this way?
isn't it better to add input file so it can run offline locally and on the ci as well?

Reply:
lets discuss

Reply:
fixed